core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -238,12 +238,34 @@ class Main {
                     info("TCP timeouts: $timeouts")
                 }
 
+                // The number of covered targets and the number of potential faults will be shown in both
+                // blackbox and whitebox
+                val elementsPair = data.find{it -> it.header == "coveredTargets"}
+
+                if (elementsPair != null) {
+                    info("Covered targets (lines, branches, faults, etc.): ${elementsPair.element}")
+                }
+                else {
+                    warn("coveredTargets not calculated in statistics")
+                }
+
+                // find the number of potential faults
+                val faultsPair = data.find{it -> it.header == "potentialFaults"}
+
+                if (faultsPair != null) {
```

Comment:
this was already computed in line 231. why again here?

Reply:
Thank you so much Andrea for your comment. Since we just need "potentialFaults", it is calculated before getting inside the if condition which checks if the test is black-box. I removed those lines since there is no need to calculate it again. In my fix, I commented out line 231, which was calculating the number of potential faults. Now, I activated that line back.